pipelines/skylab/build_indices/BuildIndices.wdl

```diff
@@ -355,7 +355,7 @@ task BuildIntervalList {
   >>>
 
   runtime {
-    docker: "quay.io/humancellatlas/secondary-analysis-umitools:0.0.1"
+    docker: "us.gcr.io/broad-gotc-prod/samtools:1.0.0-1.11-1624651616"
```

Comment:
Let's rollback this change so that we don't publish a new version of the pipeline that doesn't run